examples/Scaffolding/DevOps/helperMethods.ps1

```diff
@@ -0,0 +1,64 @@
+function createProject{
+    param(
+        [String]$org,
+        [String]$projectName,
+        [String]$process
+    )
```

Comment:
Can add source_control and visibility parameters as inputs for completeness. even if you default to git and private it is fine.